README.md

```diff
@@ -1,4 +1,4 @@
-# droidlet
+# droidlet <img style="float: right;" src="https://drive.google.com/uc?export=view&id=11tx9ZoQ9bP8SryqITN7wBP5cKOmMtS2I" width="300"/> </br></br>
```

Comment:
Just get rid of the 'droidlet' text entirely so it only says droidlet only once?